laa/iot_devices/demo_integrated.py

```
@@ -0,0 +1,132 @@
+    print("  - Net power array tracking for all devices")
+    print("  - Pool pump integration\n")
+    
+    # Initialize system
+    print("Initializing IEEE 39-Bus System with IoT Devices...")
+    ieee39 = MockIEEE39()
+    integrator = IoTLAAIntegrator(ieee39)
+    
+    # Deploy devices with N=1 (can be changed to N=2, N=3, etc.)
+    N = 1
+    print(f"\nDeploying IoT devices with scaling factor N={N}...")
+    device_count = integrator.deploy_iot_devices(N=N)
```

Ruff (0.13.3)

Local variable `device_count` is assigned to but never used

Remove assignment to unused variable `device_count`

(F841)